revenue_app/urls.py

```diff
@@ -11,6 +11,7 @@
     TopOrganizersLatam,
     top_events_json_data,
     top_organizers_json_data,
+    dashboard_summary,
```

Comment:
please order this and put everything in camel case notation

Reply:
the CamelCase ones are Class Based Views, and the snake_case ones are Function Based Views

Reply:
gotcha, thanks, just apply the order in that case